.github/workflows/main-ci.yml

```
@@ -0,0 +1,280 @@
+  # ==========================================================================
+  # STAGE 1: Fast Checks (Run in Parallel) - ~2 minutes
+  # ==========================================================================
+
+  quick-validation:
+    name: Quick Validation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Validate JSON files
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2162:info:2:61: read without -r will mangle backslashes

(shellcheck)